front/components/workspace/ProviderManagementModal.tsx

```diff
@@ -173,6 +173,9 @@ export function ProviderManagementModal({
             <ContextItem.List>
               {MODEL_PROVIDER_IDS.map((provider) => {
                 const LogoComponent = MODEL_PROVIDER_LOGOS[provider];
+                if (!modelProviders[provider]) {
+                  return null;
+                }
```

Comment:
Can't we just skip the `modelProviders[provider].join(", ")` if no model is found for this provider ? I'm not sure what we want to do for deepseek, but the rest of the code should be ok, so we can enable/disable it if needed

Reply:
But @tdraier why would we want to display the Deepseek entries to an admin if no models are available to them anyway?


I could display something like that (instead of not displaying the provider as I did) but why do you think it's better to display an empty provider?

![Screenshot 2024-12-30 at 15 19 53](https://github.com/user-attachments/assets/7a8e3398-f843-4695-a13d-4bb83efdc291)